app/src/main/java/com/example/lhi06/residue/FindAlbumActivity.java

```diff
@@ -39,6 +41,15 @@ protected void onCreate(@Nullable final Bundle savedInstanceState) {
         recyclerView.setAdapter(adapter);
         service = new ItunesService();
 
+        adapter.setAlbumClickListener(new AlbumAdapter.AlbumClickListener() {
+            @Override
+            public void onClick(@NonNull final Album album) {
+                Intent i = new Intent(FindAlbumActivity.this, AlbumReviewActivity.class);
+                i.putExtra("album", album);
```

Comment:
Android advises that extra names contain the app package. So yours could be something like `"com.example.lhi06.residue.EXTRA_ALBUM"`